ci: flaky Cypress test for oas3-multiple-media-types

<!---
  Thanks for filing a bug report! 😄

  Before you submit, please read the following:

  If you're here to report a security issue, please STOP writing an issue and
  contact us at security@swagger.io instead!

  Search open/closed issues before submitting!

  Issues on GitHub are only related to problems of Swagger-UI itself. We'll try
  to offer support here for your use case, but we can't offer help with projects
  that use Swagger-UI indirectly, like Springfox or swagger-node.

  Likewise, we can't accept bugs in the Swagger/OpenAPI specifications
  themselves, or anything that violates the specifications.
-->

### Q&A (please complete the following information)

see notes below on existing Cypress test.

### Content & configuration
<!--
  Provide us with a way to see what you're seeing,
  so that we can fix your issue.
-->

see notes below on existing Cypress test.

### Describe the bug you're encountering
<!-- A clear and concise description of what the bug is. -->

Recently, running CI on both Jenkins and GA, 
```
1) OpenAPI 3.0 Multiple Media Types with different schemas
       multipart/form-data (only 'bar')
         should execute multipart/form-data:
     AssertionError: Timed out retrying: Expected to find element: `.responses-wrapper .curl-command`, but never found it.
      at Context.eval (http://localhost:3230/__cypress/tests?p=test/e2e-cypress/tests/features/oas3-multiple-media-type.js:123:50)
```

### To reproduce...

Steps to reproduce the behavior:
1. Go to Cypress, `oas3-multiple-media-types.js` test and run it.
2. The first test sometimes fails

### Expected behavior
<!-- A clear and concise description of what you expected to happen. -->

This test file should always pass

### Screenshots
<!-- If applicable, add screenshots to help explain your problem. -->

### Additional context or thoughts
<!-- Add any other context about the problem here. -->

likely async timing issue.

--- a/test/e2e-cypress/tests/features/oas3-multiple-media-type.js
+++ b/test/e2e-cypress/tests/features/oas3-multiple-media-type.js
@@ -30,10 +30,10 @@ describe("OpenAPI 3.0 Multiple Media Types with different schemas", () => {
     it("should execute multipart/form-data", () => {
       cy.get("@selectMediaType")
         .select(mediaTypeFormData)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      // cURL component
-      cy.get(".responses-wrapper .curl-command")
+        // cURL component
+        .get(".responses-wrapper .curl-command")
         .should("exist")
         .get(".responses-wrapper .curl-command span")
         .should("contains.text", "bar")
@@ -42,14 +42,14 @@ describe("OpenAPI 3.0 Multiple Media Types with different schemas", () => {
     it("should execute application/x-www-form-urlencoded THEN execute multipart/form-data", () => {
       cy.get("@selectMediaType")
         .select(mediaTypeUrlencoded)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      cy.get("@selectMediaType")
+        .get("@selectMediaType")
         .select(mediaTypeFormData)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      // cURL component
-      cy.get(".responses-wrapper .curl-command")
+        // cURL component
+        .get(".responses-wrapper .curl-command")
         .should("exist")
         .get(".responses-wrapper .curl-command span")
         .should("contains.text", "bar")
@@ -58,14 +58,14 @@ describe("OpenAPI 3.0 Multiple Media Types with different schemas", () => {
     it("should execute application/json THEN execute multipart/form-data", () => {
       cy.get("@selectMediaType")
         .select(mediaTypeJson)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      cy.get("@selectMediaType")
+        .get("@selectMediaType")
         .select(mediaTypeFormData)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      // cURL component
-      cy.get(".responses-wrapper .curl-command")
+        // cURL component
+        .get(".responses-wrapper .curl-command")
         .should("exist")
         .get(".responses-wrapper .curl-command span")
         .should("contains.text", "bar")
@@ -77,10 +77,10 @@ describe("OpenAPI 3.0 Multiple Media Types with different schemas", () => {
     it("should execute application/x-www-form-urlencoded", () => {
       cy.get("@selectMediaType")
         .select(mediaTypeUrlencoded)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      // cURL component
-      cy.get(".responses-wrapper .curl-command")
+        // cURL component
+        .get(".responses-wrapper .curl-command")
         .should("exist")
         .get(".responses-wrapper .curl-command span")
         .should("contains.text", "foo")
@@ -89,14 +89,14 @@ describe("OpenAPI 3.0 Multiple Media Types with different schemas", () => {
     it("should execute multipart/form-data THEN execute application/x-www-form-urlencoded", () => {
       cy.get("@selectMediaType")
         .select(mediaTypeFormData)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      cy.get("@selectMediaType")
+        .get("@selectMediaType")
         .select(mediaTypeUrlencoded)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      // cURL component
-      cy.get(".responses-wrapper .curl-command")
+        // cURL component
+        .get(".responses-wrapper .curl-command")
         .should("exist")
         .get(".responses-wrapper .curl-command span")
         .should("contains.text", "foo")
@@ -105,14 +105,14 @@ describe("OpenAPI 3.0 Multiple Media Types with different schemas", () => {
     it("should execute application/json THEN execute application/x-www-form-urlencoded", () => {
       cy.get("@selectMediaType")
         .select(mediaTypeJson)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      cy.get("@selectMediaType")
+        .get("@selectMediaType")
         .select(mediaTypeUrlencoded)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      // cURL component
-      cy.get(".responses-wrapper .curl-command")
+        // cURL component
+        .get(".responses-wrapper .curl-command")
         .should("exist")
         .get(".responses-wrapper .curl-command span")
         .should("contains.text", "foo")
@@ -126,12 +126,12 @@ describe("OpenAPI 3.0 Multiple Media Types with different schemas", () => {
       // final curl should have both "bar" and "foo"
       cy.get("@selectMediaType")
         .select(mediaTypeJson)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      // cURL component
-      cy.get(".responses-wrapper .curl-command")
+        // cURL component
+        .get(".responses-wrapper .curl-command")
         .should("exist")
         .get(".responses-wrapper .curl-command span")
         .should("contains.text", "foo")
@@ -140,14 +140,14 @@ describe("OpenAPI 3.0 Multiple Media Types with different schemas", () => {
     it("should execute multipart/form-data THEN execute application/json", () => {
       cy.get("@selectMediaType")
         .select(mediaTypeFormData)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      cy.get("@selectMediaType")
+        .get("@selectMediaType")
         .select(mediaTypeJson)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      // cURL component
-      cy.get(".responses-wrapper .curl-command")
+        // cURL component
+        .get(".responses-wrapper .curl-command")
         .should("exist")
         .get(".responses-wrapper .curl-command span")
         .should("contains.text", "foo")
@@ -157,14 +157,14 @@ describe("OpenAPI 3.0 Multiple Media Types with different schemas", () => {
       // final curl should have both "bar" and "foo"
       cy.get("@selectMediaType")
         .select(mediaTypeUrlencoded)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      cy.get("@selectMediaType")
+        .get("@selectMediaType")
         .select(mediaTypeJson)
-      cy.get("@executeBtn")
+        .get("@executeBtn")
         .click()
-      // cURL component
-      cy.get(".responses-wrapper .curl-command")
+        // cURL component
+        .get(".responses-wrapper .curl-command")
         .should("exist")
         .get(".responses-wrapper .curl-command span")
         .should("contains.text", "foo")

